git-open-pull.go

```diff
@@ -162,12 +195,14 @@ func main() {
 	fmt.Printf("Issue: %d (%s)\n", issueNumber, *issue.Title)
 	head := fmt.Sprintf("%s:%s", settings.User, branch)
 	fmt.Printf("pulling from %s into %s/%s branch %s\n", head, settings.BaseAccount, settings.BaseRepo, settings.BaseBranch)
-	yn, err := input.Ask("confirm [y/n]", "")
-	if err != nil {
-		log.Fatal(err)
-	}
-	if yn != "y" {
-		log.Fatal("exiting")
+	if *interactive {
+		yn, err := input.Ask("confirm [y/n]", "")
+		if err != nil {
+			log.Fatal(err)
+		}
+		if yn != "y" {
```

Comment:
Unrelated to this PR but thing that's always bugged me is I've hit this case if I put `Y`. Could/should we add a capital as a valid input here?